solutions/devsprint-2/DeliveryAppChallenge/Screens/Settings/SettingsViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SettingsViewModel.swift
+//  DeliveryAppChallenge
+//
+//  Created by Pedro Boga on 17/11/21.
+//
+
+import Foundation
+import UIKit
+
+class SettingsViewModel {
+    
+    func getSavedName() -> String {
+        guard let key = Keys(rawValue: Keys.name.rawValue) else { return "" }
+        let name = DataSourceUserDefaults.getValue(with: key) as! String
```

Comment:
Um ponto de atenção:
Nunca é bom forçar as variáveis 😅
Ideal é sempre verificar se existe para evitar crashes